### PR TITLE
refactor(test): Properly split up joined paths

### PR DESCRIPTION
**Pull request by jhermsmeier**
Change-Type: patch

### CODE REVIEW

File: test/parse.js [hunk 1]
```diff
@@ -18,15 +18,15 @@ describe( 'BlockMap.parse()', function() {
   context( 'when input is a string', function() {
     it( 'parses valid data', function() {
       var json = require( `./data/version-2.0` )
-      var xml = fs.readFileSync( path.join( __dirname, `/data/version-2.0.bmap` ) )
+      var xml = fs.readFileSync( path.join( __dirname, 'data', `version-2.0.bmap` ) )
```
Comment (zvin) [inline test/parse.js]: there's no need for backticks here

File: test/parse.js [hunk 2]
```diff
@@ -18,15 +18,15 @@ describe( 'BlockMap.parse()', function() {
...
       assert.deepEqual( json, BlockMap.parse( xml ) )
     })
   })
 
   context( 'when input is a buffer', function() {
     it( 'parses valid data', function() {
       var json = require( `./data/version-2.0` )
-      var xml = fs.readFileSync( path.join( __dirname, `/data/version-2.0.bmap` ) )
+      var xml = fs.readFileSync( path.join( __dirname, 'data', `version-2.0.bmap` ) )
```
Comment (zvin) [inline test/parse.js]: backticks

**Review by zvin — APPROVED** [review]

File: test/parse.js (hunk 1 above)
Comment (jhermsmeier) [reply test/parse.js]: Mended!

File: test/parse.js (hunk 2 above)
Comment (jhermsmeier) [reply test/parse.js]: Fixed

**Pull request merged**